test/TritonGPU/tritongpu_ops.mlir

```diff
@@ -1,11 +0,0 @@
-// RUN: triton-opt %s | triton-opt | FileCheck %s
-
-#shared0 = #ttg.shared<{vec = 8, perPhase = 4, maxPhase = 2, order = [1, 0], CTAsPerCGA = [1, 1], CTASplitNum = [1, 1], CTAOrder = [1, 0], hasLeadingOffset=true}>
-
-module attributes {"ttg.target" = "cuda:0", "ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 1 : i32, "ttg.threads-per-warp" = 32 : i32} {
```

Comment:
I'm unsure about what this file is testing? I'll restore if needed

Reply:
the point is to test parsing/printing individually. In general it is convient when adding ops/types and makes good IR example. If you prefer to remove it that's fine with me but I think keeping it makes sense

Reply:
Got it. Let me just move it to `ops.mlir`